packages/suite/src/components/suite/Preloader/SuiteLayout/SuiteLayout.tsx

```diff
@@ -73,7 +73,7 @@ export const ContentWrapper = styled.div`
     flex: 1;
     width: 100%;
     max-width: ${MAX_CONTENT_WIDTH};
-    padding: ${spacingsPx.lg} ${HORIZONTAL_LAYOUT_PADDINGS} 90px ${HORIZONTAL_LAYOUT_PADDINGS};
+    padding: ${spacingsPx.lg} ${HORIZONTAL_LAYOUT_PADDINGS} 134px ${HORIZONTAL_LAYOUT_PADDINGS};
```

Comment:
It does! But the issue is not caused by the `SuiteLayout`'s incorrect paddings, it is caused by the `PromoBanner`'s `absolute` positioning making the bottom layout padding look smaller ;) 

Let's say the padding was even taller and you increase the global layout bottom padding even more to have room for it on the Dashboard. Then it might so happen that the bottom padding become too large in other places, causing another bug.

 It is not the case now but it's an explanation of why I decided to leave the original comment 🙏